.github/workflows/ci-build.yml

```diff
@@ -228,10 +228,10 @@ jobs:
           submodules: 'recursive'
           fetch-depth: 0
 
-      - name: Set up Python 3.9
+      - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.9'
+          python-version: '3.13'
```

Comment:
This is applying the version change to the docs build, not the sdist as tested.

Testing the docs builds in CI, especially since we are starting to work on typing and parsing scripts, might be a good idea. I wouldn't be surprised if it fails with other versions of packages, sphinx version 2 is kinda old, they are at version 8.

Reply:
You're right, thanks for catching that!